index.css

```diff
@@ -0,0 +1,43 @@
+.main {
+  margin-top: 2rem;
+  display: flex;
+}
+
+.main section {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  flex-basis: 50%;
+  padding: 3rem;
+}
+
+.main section .brand-name {
+  color: var(--primary-color-dark);
+  font-size: 2.5rem;
+  font-weight: bold;
+  font-family: 'Quintessential', cursive;
+  letter-spacing: 0.5rem;
+}
+
+.main .main__intro-text {
+  font-weight: 500;
+  margin-top: 2rem;
+}
+
+.main .main__intro-subtext {
+  font-weight: 500;
```

Comment:
you have used values here , be consistent with the type of values you are using

Reply:
I use generally font-weight as a predefined value with bold but sometimes it is not desirable so I change that with specific font-weight